kvmagent/kvmagent/plugins/hygon_device_plugin.py

```
@@ -558,8 +567,106 @@ def generate_hygon_mdev_devices(self, req):
         mdev_bindings = self._collect_mdev_bindings(device_idx_to_pci_bdf)
         rsp.mdevBindings = mdev_bindings
 
+        # Execute start_qemu.py --update_qemu_conf to add VFIO devices to qemu.conf
+        r, o, e = bash_roe("timeout 60 python3 %s --update_qemu_conf" % self.HCT_START_QEMU_SCRIPT)
```

Ruff (0.14.8)

`bash_roe` may be undefined, or defined from star imports

(F405)